port_forwarding.go

```diff
@@ -0,0 +1,335 @@
+package testcontainers
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net"
+	"time"
+
+	"github.com/docker/docker/api/types/container"
+	"golang.org/x/crypto/ssh"
+
+	"github.com/testcontainers/testcontainers-go/internal/core/network"
+)
+
+const (
+	image string = "testcontainers/sshd:1.2.0"
+	// hostInternal is the internal hostname used to reach the host from the container,
+	// using the SSHD container as a bridge.
+	hostInternal string = "host.testcontainers.internal"
```

Comment:
```suggestion
	// HostInternal is the internal hostname used to reach the host from the container,
	// using the SSHD container as a bridge.
	HostInternal string = "host.testcontainers.internal"
```

Maybe make it public so that you can use it easily from the modules?

```
testcontainers.HostInternal
```

Reply:
I like it, let's do it, thanks!